src/DefaultUnleash.php

```diff
@@ -59,21 +63,157 @@ public function isEnabled(string $featureName, ?Context $context = null, bool $d
             }
         }
 
-        return $this->isFeatureEnabled($feature, $context, $default)->isEnabled();
+        return $this->isFeatureEnabled($feature, $context, $dependencies, $default)->isEnabled();
```

Comment:
why can't dependencies be taken from the feature and need to be passed as arguments